contracts/evoting/types/admin.go

```diff
@@ -44,6 +45,51 @@ func (af AdminForm) Deserialize(ctx serde.Context, data []byte) (serde.Message,
 	return message, nil
 }
 
+// AddAdmin add a new admin to the system.
+func (adminForm *AdminForm) AddAdmin(userID string) error {
+	sciperInt, err := strconv.Atoi(userID)
+	if err != nil {
+		return xerrors.Errorf("Failed to convert SCIPER to an INT: %v", err)
+	}
+
+	adminForm.AdminList = append(adminForm.AdminList, sciperInt)
+
+	return nil
+}
+
+// IsAdmin return the index of admin if userID is one, else return -1
```

Comment:
I guess it's not hugely problematic in this case, but in general you would always try to differ between your code executing correctly and raising an error due to unexpected input/behaviour

Reply:
also when a function is called `IsAdmin` I would expect it to return True/False/Error, not an index

I would rename it to `getAdminIndex` or something similar